packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -554,13 +555,15 @@ class FadeTransition extends SingleChildRenderObjectWidget {
     return RenderAnimatedOpacity(
       opacity: opacity,
       alwaysIncludeSemantics: alwaysIncludeSemantics,
+      advisoryDevicePixelRatio: MediaQuery.maybeOf(context)?.devicePixelRatio ?? 1.0,
```

Comment:
I don't think we need this - just at paint/composition time check `ui.window.devicePixelRatio` right?

Reply:
(I guess I'm wrong about this, since if DRP changes we'd want to get updated, so subscribing here woul dbe more appropriate)